Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/ViewBookmarkLoaderInterface.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <Prefab/ViewBookmarkComponent.h>
+
+/*!
+ * ViewBookmarkIntereface
+ * Interface for saving/loading View Bookmarks.
+ */
+namespace AzToolsFramework
+{
+    class ViewBookmarkLoaderInterface
```

Comment:
An API comment for this would be good (I'd move the one above from the namespace to this type and just use `//!` style, much easier to maintain